src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Helpers/DictionaryHelpers.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NewRelic.Agent.Extensions.Helpers
+{
+    public static class DictionaryHelpers
+    {
+        public static IReadOnlyDictionary<string, object> FromJson(string json)
```

Comment:
Codecov wants a unit test for this helper...

Reply:
I am working on a test.